### ISSUE TITLE
Tune hyperparameters

### ISSUE THREAD

**Issue by maciejzj** [hand-drawn-markers-polsl/marker-recognition#23]
* Dropout rate
* Batch size
* Conv kernel size
* Image size
* Learning rate/momentum
* Fully connected units
* Augmentation operations

Hyper-parameter optimization is TensorBoard built-in, relate to [this](https://www.tensorflow.org/tensorboard/hyperparameter_tuning_with_hparams).

Alternatively we can use RayTune, relate  to [this tutorial](https://towardsdatascience.com/hyperparameter-tuning-with-keras-and-ray-tune-1353e6586fda).
